brownie-config.yaml

```diff
@@ -15,5 +15,5 @@ networks:
       default_balance: 10000000
       evm_version: istanbul
       mnemonic: 'rib diamond horse main tenant habit nothing fish alarm budget age comic filter execute used'
-      # fork: http://localhost:9545@14042429
-      fork: https://mainnet.infura.io/v3/_your_api_key_@14042429
+      # fork: http://localhost:9545@17965130
+      fork: https://mainnet.infura.io/v3/_your_api_key_@17965130
```

Comment:
Can't we use an up-to-date fork each time?

Reply:
remove block number from config